3주차 멘토링 PR입니다.

sete3683 opened the pull request:
감사합니다.

kingyj00 committed:
[Feat] CalendarEntity

kingyj00 committed:
[Feat] CalendarEntity

kingyj00 committed:
[Feat] CalendarEntity

kingyj00 committed:
[Feat] CalendarController

kingyj00 committed:
[Feat] CalendarService

kingyj00 committed:
[Feat] CalendarRepository

kingyj00 committed:
[Feat] CalendarDtoRequest

kingyj00 committed:
[Feat] CalendarDtoRequest

kingyj00 committed:
[Feat] CalendarDtoResponse

kingyj00 committed:
[Feat] CalendarDtoResponse

kingyj00 committed:
[Test] CalendarTestCode

kingyj00 committed:
[Fix] CalendarEntity-Import

kingyj00 committed:
[Rename] DtoFileName

kingyj00 committed:
[Rename] DtoFileName

kingyj00 committed:
[Fix] CalendarEntity-Add-Getter,Constructor

kingyj00 committed:
[Fix] Add Getter, Setter

kingyj00 committed:
[Test] CalendarCRUDTest

# Conflicts:
#	backend/src/test/java/com/frend/planit/domain/calendar/CalendarTest.java

kingyj00 committed:
[Test] CalendarCRUDTest

kingyj00 committed:
[Rename] FixCalendarTestClassName

kingyj00 committed:
[Fix] CalendarEntity-생성201반환

kingyj00 committed:
Merge branch 'feature/C-01' of https://github.com/prgrms-web-devcourse-final-project/WEB3_4_Fr-End_BE into feature/C-01

# Conflicts:
#	backend/src/main/java/com/frend/planit/domain/calendar/controller/CalendarController.java
#	backend/src/main/java/com/frend/planit/domain/calendar/entity/CalendarEntity.java
#	backend/src/test/java/com/frend/planit/domain/calendar/CalendarTest.java

kingyj00 committed:
[Fix] CalendarDto-setter삭제,record/valid적용

kingyj00 committed:
[Fix] CalendarEntity-중복제거/제약조건

kingyj00 committed:
[Refector] CalendarService/[Feat] Exception

kingyj00 committed:
[Fix] CalendarController-AddTranscationDelete

kingyj00 committed:
[Refactor] CalendarEntity-생성자에서빌더로변경

kingyj00 committed:
[Test] CalendarControllerTest

kingyj00 committed:
[Fix] 불필요한 파일 삭제

sete3683 committed:
[Refactor] 불필요한 Response 헬퍼 삭제 (#54)

- Issue #52
- ApiResponseHelper 사용중이던 기존 코드는 @ResponseStatus로 모두 수정

sete3683 committed:
[Fix] 로깅 경로 달라지는 버그 해결 (#53)

- Issue #51
- logs/main과 logs/test 경로 구분하도록 함

kingyj00 committed:
[Feat] AccommodationEntity

kingyj00 committed:
[Feat] AccommodationController

kingyj00 committed:
[Feat] AccommodationService

kingyj00 committed:
[Feat] AccommodationDto (Request,Response)

kingyj00 committed:
[Feat] AccommodationRepository

c-hano committed:
[FIX]/U-01-01 #59 : [FEAT]U-01 병합 오류 해결 및 해당 기능 PR (#61)

이하 Squash
----------------

* [FIX] secret 올린거 제외하고 다시 이사

* [REFACTOR] yml 파일들 원레 주소로 이사

sete3683 committed:
[Refactor] 에러 타입 간소화 (#58)

- Issue #56

c-hano committed:
[FEAT] Controller Auth/User 로 분리 및 테스트 코드 재설정

c-hano committed:
[REFACTOR] Auth와 user 분리, [FEAT] refresh 토큰 발급

c-hano committed:
[FEAT] refresh 토큰 발급 처리

c-hano committed:
[FEAT] 로그아웃 기능 추가

c-hano committed:
[TEST] AuthController 테스트

c-hano committed:
[REFACTOR] securityConfig 수정, SecurityContextHolder 적용

c-hano committed:
[FEAT] refresh 토큰 값 만료 임박 시 다시 refresh 토큰 발급

c-hano committed:
[TEST] 수정사항 test 업데이트

zelly-log committed:
Feature/MB-05 #37 : 메이트 모집 게시판 댓글 CRUD 구현 (#63)

* [Remove] Delete unuse file .gitkeep

* [Feat] Add MateComment Entity

* [Feat] Add MateCommentRequestDto

* [Feat] Add profileImageUrl

* [Feat] Add MateCommentRepository

* [Feat] Add MateCommentRequestDto

* [Feat] Add MateCommentResponseDto

* [Feat] ADD ErrorType "MATE_COMMENT_NOT_FOUND"

* [Feat] ADD Annotation @Getter, @Setter

* [Feat] Add MateCommentController for comment CRUD functionality

* [Feat] Add MateCommentRepository for comment CRUD functionality

* [Feat] Add MateCommentRequestDto for comment CRUD functionality

* [Feat] Add MateCommentService for comment CRUD functionality

* [Refactor] fix default paging size 10 -> 8

* [Chore] 불필요한 주석 제거

* [Feat] Create MateCommentMapper

* [Refactor] 불필요한 Response 헬퍼 삭제 (#54)

- Issue #52
- ApiResponseHelper 사용중이던 기존 코드는 @ResponseStatus로 모두 수정
# Conflicts:
#	backend/src/main/java/com/frend/planit/domain/mateboard/post/controller/MateController.java

* [Fix] 로깅 경로 달라지는 버그 해결 (#53)

- Issue #51
- logs/main과 logs/test 경로 구분하도록 함

* [FIX]/U-01-01 #59 : [FEAT]U-01 병합 오류 해결 및 해당 기능 PR (#61)

이하 Squash
----------------

* [FIX] secret 올린거 제외하고 다시 이사

* [REFACTOR] yml 파일들 원레 주소로 이사

* [Refactor] 에러 타입 간소화 (#58)

- Issue #56

* [Refactor] fix default paging size 10 -> 8

# Conflicts:
#	backend/src/main/java/com/frend/planit/domain/mateboard/post/controller/MateController.java

* [Refactor] ApiResponseHelper 제거

---------

Co-authored-by: sete3683 <108328000+sete3683@users.noreply.github.com>
Co-authored-by: c-hano <wdq0304@naver.com>

kingyj00 committed:
[Fix] Overall AccommodationCd

c-hano committed:
[REFACTOR] AuthenticationPrincipal 로 userid값 불러오는 코드 간결화

c-hano committed:
[FEAT] 토큰 발급시 ROLE 체크하는 로직 추가

kingyj00 committed:
[Feat] NewFolder-Client/Scheduler

kingyj00 committed:
[Fix] Service에서 Entity 상속문제 해결

c-hano committed:
Merge pull request #67 from prgrms-web-devcourse-final-project/feature/U-02

Feature/U-02 #62 : logout, jwt token refresh, @AuthenticationPrincipal 이용

kingyj00 committed:
[Refactor] Add 제약조건 Entity

kingyj00 committed:
[Feat] NewEnit-TourAPI to DB ones

kingyj00 committed:
[Refactor] All-AccommodationCd

c-hano committed:
[FIX] securityConfig에 h2-console 접근 허용

kingyj00 committed:
fix: resolve merge conflict in application-test.yml

kingyj00 committed:
Merge yml From Develop

c-hano committed:
[CHORE] ignore에 db_local.mv.db 추가

c-hano committed:
Merge pull request #73 from prgrms-web-devcourse-final-project/bug/UB-01

Bug/U-01 #71 : h2 접속 해결

kingyj00 committed:
Feature/A-01 #55 : TourAPI 동기화 및 CRUD #74

kingyj00 committed:
[Feat] 가²©/이름 (오름,내림)차 기능

kingyj00 committed:
[Refactor] TourApiClient-paging(200/1)

kingyj00 committed:
Feature/A-02 #57 : 가격 및 이름 (오름,내림)순 기능 #75

Feature/A-02 #57 : 가격 및 이름 (오름,내림)순 기능

sete3683 committed:
Feature/Im-01 : 이미지 업로드 구현 (#70)

* [Feat] 이미지 업로드 기능 구현

- Issue #47
- Presigned POST URL을 사용한 이미지 업로드
- application-secret 설정 필요

* [Fix] api 엔드포인트 수정

c-hano committed:
Feature/U-03 #68 : 소셜 로그인 연동, 마이페이지 (#76)

* [FEAT] 소셜 최초 로그인 시 회원가입, 로그인 분리

* [FEAT] 마이페이지 정보 조회, 수정 DTO 작성

* [FEAT] 내 정보 수정 기능

* [FEAT] update 로직 분리, 개별 update 가능하게 dto 생성 [REFACTOR] errortype -> REQUEST_NOT_VALID 로 통일

# Conflicts:
#	backend/src/main/java/com/frend/planit/global/response/ErrorType.java

* [TEST] controller들 테스트 코드

* [FIX] 소셜 로그인 오류 해결

# Conflicts:
#	db_local.mv.db

* [CHORE] ignore trace.dv 추가

* [FEAT] KAKAO 소셜 로그인 추가

* [FEAT] NAVER 소셜 로그인 연동, 각 소셜 dto 따로 구현

hellonaeunkim committed:
Feature/S-02 #46 : 여행 일정 (Travel) CRUD 구현 (#77)

* feat : TravelEntity 생성
TravelEntity와 FK로 연결된 ScheduleEntity 추가

* feat : 전체 행선지 조회 기능 구현
- ScheduleRepository 추가
- ErrorType 추가

# Conflicts:
#	backend/src/main/java/com/frend/planit/global/response/ErrorType.java

* fix : Repository 메서트 typo 수정

* test : Controller 전체 행선지 조회 성공, 실패 테스트 코드

* refactor : ApiResponseHelper 클래스 삭제로 인한 Controller 응답코드 형태 리팩토링

* refactor : ScheduleDayEntity 도입으로 인한 TravelEntity, TravelResponse 리팩토링

* feat : ScheduleDay 도메인 도입으로 Travel 도메인 전체 구조 변경
- 전체 행선지 조회 Service Exception 추가
- Travel ErrorType 추가

# Conflicts:
#	backend/src/main/java/com/frend/planit/global/response/ErrorType.java

* feat : 행선지 생성, 삭제 기능 구현
- TravelGroupingUtils 추가 : 여행 Entity를 날짜별로 그룹화 (Request 요청시 활용)
- ErrorType 추가

# Conflicts:
#	backend/src/main/resources/application-test.yml
#	db_local.mv.db

* fix : H2 콘솔 접솔 URL 변경

* feat : 행선지 수정 기능 구현
- test/source/application.yml 수정

* test : Service 테스트 코드 작성

kingyj00 committed:
Merge From Develop

hellonaeunkim committed:
Feature/S-01 #24 : 여행 일정 (Schedule) CRUD 구현 (#82)

이하 Squash

* feat : 여행 일정 조회, 생성 기능 구현
- TravelRequest Validation 추가
- TravelEntity nullable 수정

* feat : 여행 일정 수정, 삭제 기능 구현

kingyj00 committed:
[Fix] Entity-UseBaseID

kingyj00 committed:
[Refactor] Service-All()toPageable

kingyj00 committed:
[Refactor] Controller-Listtopaging

zelly-log committed:
Feature/MB-02 #65 : 게시글 전체 조회 필터링 기능 적용 및 User 연동 (#83)

이하 Squash

* [Remove] User 로그인 연동으로 사용하지 않는 AuthorGender.java 삭제

* [Feat] QueryDSL 의존성 추가

* [Feat] 메이트 게시판 글 작성 시 에러 코드 추가

* [Feat] 모집 인원 수 필드 추가, User 연동

* [Feat] 모집 인원 수 필드 추가, User 연동

* [Feat] 모집 인원 수 필드 추가, User 연동

* [Feat] QuerydslConfig 추가

* [Feat] User 연동 및 게시글 전체 조회 필터링 적용

* [Feat] 게시글 전체 조회 필터링 적용하기 위해 QueryDSL 사용하여 구현

sete3683 committed:
Feature/Im-02 : 이미지 조회 기능 추가 (#80)

이하 Squash

* [Feat] 이미지 조회 기능 추가

* [Feat] Response Dto 작성 및 적용

kingyj00 committed:
[Feat] AddErrorType

sete3683 committed:
Feature/Im-03 : 이미지 교체 기능 추가 (#81)

이하 Squash

* [Feat] 이미지 조회 기능 추가

* [Feat] 이미지 변경 기능 추가

kingyj00 committed:
[Remove] ControllerTest-기능구현완료후 다시구현예정

kingyj00 committed:
[Fix] DB SAVE 0 Issue

kingyj00 committed:
[Refactor] Add AreaCode,Cat,MapXY

kingyj00 committed:
:wqMerge branch 'develop' of https://github.com/prgrms-web-devcourse-final-project/WEB3_4_Fr-End_BE into feature/C-01
Sync with latest develop updates#
Sync with latest develop updates# the commit.Sync with latest develop updatesSync with latest develop updatesSync with latest develop updatesSync with latest develop updatc

Merge branch 'develop' into feature/C-01
Sync with latest develop updates.

kingyj00 committed:
Merge branch 'develop' into feature/C-01

kingyj00 committed:
[Fix] Delete Log,db_local

kingyj00 committed:
[Refactor] TourApiDto 유효성 검증 및 불변 설계

kingyj00 committed:
[Fix] Change Amenities

kingyj00 committed:
Feature/A-02 #57 : 지역,숙소분류,좌표 동기화 DB저장 수정 #87

Feature/A-02 #57 : 지역,숙소분류,좌표 동기화 DB저장 수정

kingyj00 committed:
[Feat] AddTourApiItems

kingyj00 committed:
[Fix] DB저장 시, Null 이슈 해결

Feature/A-02 #57 : DB저장 시, Null 이슈 해결

zelly-log committed:
Feature/MB-03 #64: 메이트 모집 기능 구현 (#89)

이하 Squash

* [Feat] MateApplication ErrorType 추가

* [Feat] 사용자 프로필 이미지, 게시글 이미지 추가

* [Feat] 신청 모집 기능 구현

* [Feat] 사용자 프로필 이미지, 게시글 이미지 추가

hellonaeunkim committed:
Refactor/S-01 #84 : Schedule 날짜 변경 시 ScheduleDay 자동 동기화 기능 추가 (#85)

이하 Squash

* feat : ScheduleEntity starDate & endDate 수정 시 ScheduleDayEntity와 TravelEntity에 동기화 되도록 로직 수정
- Schedule Controller, Service 메서드 이름 변경
- DailyTravelResponse에 scheduleDayId 추가하여 프론트에서 날짜별 행선지 (Travel) 식별 및 요청 처리 가능하도록 개선
- DailyTravelResponse에 scheduleDayId 추가에 따른 TravelGroupingUtils 메서드 로직 변경

* refactor : DailyTravelResponse 수정으로 인한 테스트 코드 리팩토링

* feat : .gitinore 추가

* feat : conflict 해결

Account-4-Review commented:
리뷰 방향성 미작성으로, 임의대로 리뷰합니다. 참고해주세요.

sete3683 committed:
Feature/Im-04 #79: 이미지 삭제 기능 구현 (#92)

이하 Squash

* [Feat] 삭제 기능 초안 작성

* [Feat] S3 파일 삭제 기능 메서드 작성

* [Feat] 이미지 정기 삭제 기능 추가

* [Fix] 컴파일 에러 수정

sete3683 committed:
[Feat] 기본 데이터 코드 초안 작성 (#94)

kingyj00 committed:
Merge branch 'develop' of https://github.com/prgrms-web-devcourse-final-project/WEB3_4_Fr-End_BE into feature/C-01
Sync with latest develop updates#
Sync with latest develop updates# the commit.Sync with latest develop updatesSync with latest develop updatesSync with latest develop updatesSync with latest develop updatc
develop 브랜치 최신 변경사항 반영
esc
end
Merge branch 'develop' into feature/C-01
develop 브랜치의 최신 변경사항을 feature/C-01 브랜치에 반영

kingyj00 committed:
[Refactor] Paging, Response, TamporaryNull

kingyj00 committed:
Merge branch 'develop' of https://github.com/prgrms-web-devcourse-final-project/WEB3_4_Fr-End_BE into feature/A-02
the commit.

c-hano committed:
[REFACTOR] security 접근 권한 수정 (#97)

kingyj00 committed:
[Refactor] Paging, Response, TamporaryNull (#98)

kingyj00 committed:
Merge branch 'develop' of https://github.com/prgrms-web-devcourse-final-project/WEB3_4_Fr-End_BE into feature/C-01

kingyj00 committed:
[Refactor] Calendar-CRUD

kingyj00 committed:
[Refactor] Calendar-CRUD

Feature/C-01 #25 : CRUD 및 로직 개선
#99

kingyj00 committed:
[Refactor] Calendar-page,DeleteGetAll

kingyj00 committed:
[Refactor] Calendar-page,DeleteGetAll

Merge pull request #100 from prgrms-web-devcourse-final-project/feature/C-01

kingyj00 committed:
[Feat] InviteEntity

kingyj00 committed:
[Feat] InviteRepository

kingyj00 committed:
[Feat] InviteService

kingyj00 committed:
[Feat] InviteController

kingyj00 committed:
[Feat] SharedCalendarEntity

kingyj00 committed:
[Feat] SharedCalendarRepository

kingyj00 committed:
[Feat] SharedCalendarService

kingyj00 committed:
[Refactor] InviteController-ShareByUserId

kingyj00 committed:
[Refactor] SharedCalendarService-ListInvite

kingyj00 committed:
[Refactor] CalendarController-AddShareList

kingyj00 committed:
[Feat] ShareCalendar-auth, Verification

kingyj00 committed:
[Fix] Service-hasModifyAccessError

kingyj00 committed:
[Fix] UserError

kingyj00 committed:
[Feat] 캘린터 초대 및 공유 기능

Merge pull request #101 from prgrms-web-devcourse-final-project/feature/C-02

kingyj00 committed:
Merge branch 'develop' of https://github.com/prgrms-web-devcourse-final-project/WEB3_4_Fr-End_BE into feature/A-02

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/accommodation/client/TourApiClient.java:
```diff
@@ -0,0 +1,96 @@
+                        .build(true)
+                        .toUri();
+
+                String xml = restTemplate.getForObject(uri, String.class);
```
`MappingJackson2XmlHttpMessageConverter` 를 MessageConverter로 등록하면, 이렇게 안 해도 알아서 객체 매핑이 될텐데요.

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/accommodation/client/TourApiClient.java:
```diff
@@ -0,0 +1,96 @@
+                        .toUri();
+
+                String xml = restTemplate.getForObject(uri, String.class);
+                log.info("📦 [TourAPI] (page {}) 응답 XML:\n{}", page, xml);
```
info x

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/accommodation/controller/AccommodationController.java:
```diff
@@ -0,0 +1,82 @@
+import com.frend.planit.domain.accommodation.service.AccommodationService;
+import com.frend.planit.global.exception.ServiceException;
+import com.frend.planit.global.response.ErrorType;
+import com.frend.planit.global.response.PageResponse; // ✅ 추가된 import
```
??

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/accommodation/controller/AccommodationController.java:
```diff
@@ -0,0 +1,82 @@
+    ) {
+        validateAdmin(admin);
+        AccommodationResponseDto created = accommodationService.create(dto);
+        URI location = URI.create("/api/accommodations/" + created.id());
```
HTTP Specification 에 따르면, CREARTED 는 **생성된 자원의 위치나 자원의 값 중 하나를 명시** 하도록 되어 있습니다.
body 를 반환했으면 굳이 URI을 던질 필요가 없어요.

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/accommodation/controller/AccommodationController.java:
```diff
@@ -0,0 +1,82 @@
+        return ResponseEntity.noContent().build();
+    }
+
+    private void validateAdmin(boolean isAdmin) {
```
역할이 정확히 어떻게 되는거죠?

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/auth/client/GoogleOauthClient.java:
```diff
@@ -0,0 +1,85 @@
+@RequiredArgsConstructor
+public class GoogleOauthClient implements OAuthClient {
+
+    @Value("${spring.security.oauth2.client.registration.google.client-id}")
```
`@ConfigurationProperites`

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/calendar/controller/CalendarController.java:
```diff
@@ -3,48 +3,93 @@
+    public ResponseEntity<CalendarResponseDto> updateCalendar(@PathVariable Long id,
+                                                              @RequestParam Long userId,
+                                                              @Valid @RequestBody CalendarRequestDto requestDto) {
+        User user = userRepository.findById(userId)
```
`@PlanItUser User user)` <- 이렇게 묶어서 해도 괜찮을 듯 해요.

Account-4-Review replied:
https://hudi.blog/spring-argument-resolver/

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/calendar/invite/entity/InviteEntity.java:
```diff
@@ -0,0 +1,38 @@
+
+    @Column(nullable = false)
+    private boolean valid;
+
```
만료기간이 있으면 좋을듯 하네요.

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/calendar/schedule/day/entity/ScheduleDayEntity.java:
```diff
@@ -0,0 +1,75 @@
+    }
+
+    // Test Code에서 사용하기 위한 setter
+    public void setId(Long id) {
```
`ReflectionTestUtils` 사용

Account-4-Review commented on backend/src/main/java/com/frend/planit/domain/calendar/service/CalendarPermissionValidator.java:
```diff
@@ -0,0 +1,25 @@
+
+@Component
+@RequiredArgsConstructor
+public class CalendarPermissionValidator {
```
나~~~~~~~~~~~~~~~~~~~~~~~중에는 캐싱을 적용해 보세요.
해당 지점의 데이터만 캐싱을 해도 효과를 많이 볼 것 같아요.

kingyj00 committed:
[Refactor] CrawlingAllArea, DeleteLog

kingyj00 committed:
[Refactor] Accommodation CrawlingAllArea, DeleteLog


Merge pull request #103 from prgrms-web-devcourse-final-project/feature/A-02

hellonaeunkim committed:
Feature/CB-01 #69 : AI 챗봇 기능 구현 (#102)

이하 Squash

* feat : 챗봇 Controller, Entity, Service, Repository 생성

* test : 채팅방 생성 테스트 코드 작성
- 엔드포인트 변경

* feat : SSE 방식 채팅방 대화 생성 및 응답 기능 구현
- AI 메세지 생성용 프롬프트 구성 및 비동기 저장 기능 구현
- 비동기 기능 사용을 위한 ThreadPoolTaskExcutor 추가
- 채팅방 대화 스트리밍 기능 및 비동기 저장 처리 구현
- Async 비동기 코드 테스트를 위한 build.gradle 의존성 추가
- AI Chat ErrorType 추가

# Conflicts:
#	backend/src/main/java/com/frend/planit/global/response/ErrorType.java

* feat : 로그인 사용자 채팅방 전체 목록 조회 및 메세지 기록 조회 기능 구현

* feat : 로그인 사용자 채팅방 삭제 기능 구현

* refactor : ErrorType 중복 제거

* feat : 회원 정보 (UserEntity) 기반 여행 일정 (Travel) 안내 메세지 응답 기능 구현
- 챗봇에게 Travel 데이터 입력 (AIUserContextHelper.java)

zelly-log committed:
Refactor/MB-01 #91: 메이트 게시판 테스트 작성 (#104)

이하 Squash

* [Feat] ADD QueryDSL

* [Feat] 댓글 사용자 이미지 기능 추가

* [Feat] 이미지 수정 기능 구현

* [Remove] 사용하지 않는 파일 삭제

* [Feat] 테스트 환경 설정

* [Feat] MateBoard 테스트를 위한 dummy data 생성

* [Feat] MateBoard Post 테스트

* [Feat] MateBoard Comment 테스트

* [Feat] MateBoard Application 테스트

* [Feat] Querydsl 설정값 추가

* [Refactor] user_id -> writer_id 컬럼명 변경

* [Refactor] created_at, updated_at 컬럼명 명시

* [Refactor] mate_post_id -> mate_id 컬럼명 변경

* [Refactor] 누락된 애너테이션 추가 @AllArgsConstructor, @NoArgsConstructor

* [Refactor] bio 추가하여 데이터 조회 쿼리 수정

* [Refactor] 게시글 이미지 없을 때 에러 수정

* [Refactor] QueryProjection 수정

* [Refactor] @Builder 추가

sete3683 committed:
[Fix] 컴파일 에러 수정

sete3683 committed:
[Fix] build.gradle 문법 오류 수정

c-hano committed:
Feature/U-04 #95 : 백엔드 -> 프론트 redirect 전달 (#106)

이하 Squash

* [REFACTOR] 브랜치 이동으로 인한 중간 커밋

* [FEAT] api 서버 거치지 않고 백에서 직접 프론트로 redirect 정보 전달

kingyj00 committed:
[Refactor] Calendar-중복성제거,간결성,가독성유지

kingyj00 committed:
MergeFromDevelop

kingyj00 committed:
[Refactor] Delete CalendarException

kingyj00 committed:
[Refactor] ChangedConnectCalendarException

kingyj00 committed:
[Refactor] Calendar-중복성제거,간결성,가독성유지

Merge pull request #111 from prgrms-web-devcourse-final-project/feature/C-02

c-hano committed:
Feature/U-05 : 로컬 로그인 회원가입 기능 추가 (#112)

* [FEAT] 로컬 로그인 기능 구현

* [FEAT] 회원가입 기능 추가

* [FIX] 자기자신은 중복검사 제외

* [TEST] 추후 테스트 할 예정

hellonaeunkim committed:
Refactor/S-01 #105 : ScheduleService 검증 로직 리팩토링 및 여행지 정렬 로직 추가 (#113)

* feat : 여행 일정 시간/분에 따른 정렬 기능 추가
- ScheduleEntity 필요없는 로직 제거
- Test Code용 Setter 제거 및 리팩토링

* feat : ScheduleService 내 일정/캘린더 검증 로직 별도 메서드로 분리
- 일정 생성시 PathVariable scheduleId 제거
- 테스트 코드 수정

zelly-log committed:
Refactor/MB-02 : 유저 연동 이슈 수정 (#116)

이하 Squash

* [Refactor] @AuthenticationPrincipal User → Long userId로 변경

* [Refactor] Test 코드 수정

* [Refactor] 변수명 통일

c-hano committed:
Refactor/RU-02 #118 : 회원가입 requestDto 수정 (#120)

* [REFACTOR] 회원가입 시 입력된 필드 값만 받아오도록 에너테이션 변경

* [CHORE] db_local.trace.db 삭제

sete3683 committed:
Feature/Im-04 #79: 이미지 리팩토링 및 테스트 코드 작성 (#117)

이하 Squash

* [Fix] 이미지 삭제 버그 수정

* [Fix] 핸들러 테스트 코드 에러 수정

* [Test] 이미지 통합 테스트 작성

* [Refactor] 이미지 트랜잭션 수정

* [Refactor] 이미지 메서드 검증용 반환값 추가

* [Fix] 적용 Profile 변경

kingyj00 committed:
Feature/C-03 #27 : LabelColor 추가 / SchemaForFE (#122)

* [Feat] ConnectWithSchedule

* [Feat] LabelColor

* [Fix] LabelColor / SchemaForFE

c-hano committed:
Feature/U-05 #107 : 내부 회원가입 로직 (#123)

* [FEAT] 로컬 로그인 기능 구현

# Conflicts:
#	backend/src/main/java/com/frend/planit/domain/auth/controller/AuthController.java
#	backend/src/main/java/com/frend/planit/domain/user/mapper/UserMapper.java
#	backend/src/main/java/com/frend/planit/domain/user/repository/UserRepository.java
#	backend/src/test/java/com/frend/planit/domain/auth/controller/AuthControllerTest.java

* [FEAT] 회원가입 기능 추가

* [TEST] 추후 테스트 할 예정

* [TEST] 테스트 코드 작성

* [TEST] UserControllerTest 작성

* [TEST] USerProfileUpdateController 테스트 코드

* [FEAT] 테스트 유저 등록

* [FEAT] 비밀번호 변경 기능 추가

* [TEST] 비밀번호 변경 테스트

zelly-log committed:
[FEAT] P-01 #108 : 결제 데이터 저장 구현 (#125)

이하 Squash

* [Refactor] @AuthenticationPrincipal User → Long userId로 변경

* [Refactor] Test 코드 수정

* [Refactor] 변수명 통일

* [Remove] 사용하지 않는 파일 삭제

* [Remove] 사용하지 않는 파일 삭제

* [Feat] Booking 테이블 생성 및 저장 기능 구현

* [Feat] Paymentlog 테이블 생성 및 저장 기능 구현

hellonaeunkim committed:
Refactor/CB-01 #114 (#126)

이하 Squash

* feat : 챗봇 이전 대화 요약 로직 삭제 및 간소화

* 소셜회원가입이 아닌 일반회원가입이 안되어서 패스워드 칼럼의 사이즈 70으로 증가, socialId 에 null 대신 공백이 추가되도록

# Conflicts:
#	backend/src/main/java/com/frend/planit/domain/user/entity/User.java

* refactor : 메세지 생성 시 Schedule이 없어도 대화 가능하도록 수정

---------

Co-authored-by: jhs512 <jangka512@gmail.com>
